### PR TITLE
Adding IN validations to the config.toml file

### PR DESCRIPTION
**Pull request by YuvHayun**
## Related Issues
fixes:

### CODE REVIEW

File: demisto_sdk/commands/validate/validators/IN_validators/IN102_is_valid_checkbox_default_field.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def fix(self, content_item: ContentTypes) -> FixResult:
                 param.name
                 in self.misconfigured_checkbox_params_by_integration[content_item.name]
             ):
-                param.required = True
+                param.required = False
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/validate/validators/IN_validators/IN102_is_valid_checkbox_default_field.py:63]: why are we flipping this validation?

File: demisto_sdk/commands/validate/validators/IN_validators/IN144_is_rep_command_contain_is_array_argument.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Integration
+
+
+class IsRepCommandContainIsArrayArgumentValidator(BaseValidator[ContentTypes]):
+    error_code = "IN144"
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/validate/validators/IN_validators/IN144_is_rep_command_contain_is_array_argument.py:19]: this isnt a new validation right? Just porting this over? seems like a strange validation

**Review by JudahSchwartz — APPROVED** [review]
looks good 🍔 . Please address my comments

File: demisto_sdk/commands/content_graph/objects/pack.py [hunk 3]
```diff
@@ -153,7 +153,17 @@ def pack_id(self) -> str:
 
     @property
     def ignored_errors(self) -> list:
-        return self.ignored_errors_dict.get(PACK_METADATA_FILENAME, [])
+        try:
+            return (
+                list(
+                    self.ignored_errors_dict.get(  # type: ignore
+                        f"file:{PACK_METADATA_FILENAME}", []
```
[the remark this answers is not in the document]
Comment (YuvHayun) [reply demisto_sdk/commands/content_graph/objects/pack.py:160]: It's not changing, it's just wasn't correct until now.

File: demisto_sdk/commands/validate/validators/IN_validators/IN102_is_valid_checkbox_default_field.py (hunk 1 above)
Comment (YuvHayun) [reply demisto_sdk/commands/validate/validators/IN_validators/IN102_is_valid_checkbox_default_field.py:63]: It was a mistake when I first added it.

File: demisto_sdk/commands/validate/validators/IN_validators/IN144_is_rep_command_contain_is_array_argument.py (hunk 2 above)
Comment (YuvHayun) [reply demisto_sdk/commands/validate/validators/IN_validators/IN144_is_rep_command_contain_is_array_argument.py:19]: Not a new one, but it is a relevant one.. we should support isArray for all rep commands.

**Pull request merged**